### DISCUSSION TITLE
Seq cluster configuration

### DISCUSSION THREAD

**Discussion by specialagentsoftware** [datalust/seq-tickets#1633] [Help] [answered]
Hello, I recently started working on installing and configuring a cluster for our prod seq environment.  I followed the instructions in the documentation. I have two EC2 instances, that are behind a load balancer with an AWS RDS postgres database as the metastore. 

I configured the seq installations to use the datastore, set the same security key, and did the node configuration to add one as the lead and the other one defaulted to being the follower as it should have, using the same key on both. When I run seq node describe. I get the expected values from both the leader and the follower. 

If I make any changes to settings or ingest any events through the load balancer, or through either node directly... those events and changes persist across all of the nodes as they are expected to. However, on the cluster page I only show the lead node. I get a message that says there is no up to date follower. If I change the follower to be the lead. The lead changes on the cluster page, but I still do not show the follower. 

I was wondering if you had any thoughts on what the issue could be. The node servers can communicate with each other and all of that. I can provide screen shots or anything that might be necessary. I just was not sure what to send you. According to the documentation.. it looks like it should. If I run seq as an application instead of a service.. I can see the lead node establish the cluster.. and I can see the follower node establish connection to the cluster. 

Thanks

**Comment by nblumhardt** [comment, marked as answer]
Hi Jeremy,

It sounds a lot like your follower is able to reach the internal API port on the leader, but not its cluster port. The docs use port `5344` as the cluster port, in the examples.

Is the cluster port open between the nodes? If so, can the follower nodes reach the leader on that port, using the hosname they're configured with? TLS can also throw up some challenges here - is the connection using `wss`?

If this doesn't help pinpoint the issue, dropping us a line via `support@datalust.co` so we can organize some time to work through it with you would be the next step.

Best regards,
Nick

**Reply by specialagentsoftware** [reply]
@nblumhardt I have sent an email with some specifics and some data as suggested. Thank you.
